products/contao

```diff
@@ -0,0 +1,38 @@
+---
+title: Contao
+category: server-app
+changelogTemplate: "https://github.com/contao/contao/blob/__RELEASE_CYCLE__/CHANGELOG.md"
```

Comment:
`changelogTemplate` may be updated with the anchor to the specific version (similar to `docker-engine.md`).

Reply:
Last version is always at the top of the document, no need to change the template to something more complex.